ChartIQDemo/ChartIQDemo/AppDelegate.swift

```diff
@@ -13,9 +13,9 @@ import ChartIQ
 class AppDelegate: UIResponder, UIApplicationDelegate {
 
     var window: UIWindow?
-    let url = "http://192.168.2.42:8080/branch/default/sample-template-native-sdk.html"
+    @objc let url = "http://yourdeployment/sample-template-native-sdk.html"
```

Comment:
Can this example be a little more clear?

Perhaps add a comment?
Perhaps use something like "http://your.deployment.example/sample-template-native-sdk.html" ?

Reply:
I'll go with your second option.